src/extensions/averagewaveforms/AverageWaveformsView/AverageWaveformsView.tsx

```diff
@@ -1,34 +1,92 @@
-import React, { FunctionComponent, useMemo } from 'react';
+import React, { FunctionComponent, useCallback, useEffect, useMemo, useState } from 'react';
+import { FaArrowDown, FaArrowUp } from 'react-icons/fa';
+import sizeMe, { SizeMeProps } from 'react-sizeme';
 import SortingUnitPlotGrid from '../../common/SortingUnitPlotGrid';
+import Splitter from '../../common/Splitter';
+import { ActionItem, DividerItem } from '../../common/Toolbars';
 import { SortingViewProps } from "../../extensionInterface";
+import AverageWaveformsToolbar from './AverageWaveformsToolbar';
 import AverageWaveformView2 from './AverageWaveformView2';
 
-const AverageWaveformsView: FunctionComponent<SortingViewProps> = (props) => {
+export type AverageWaveformAction = ActionItem | DividerItem
+
+
+const TOOLBAR_INITIAL_WIDTH = 55
+
+const AverageWaveformsView: FunctionComponent<SortingViewProps & SizeMeProps> = (props) => {
```

Comment:
Getting `SizeMe` involved to compute width so we can use the slider. We might not need to do this, but I believe it was used in the time series component set.